sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_async_lro.py

```diff
@@ -0,0 +1,89 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from urllib.parse import urlencode
+from azure.core.polling.base_polling import OperationFailed, BadStatus
+from azure.core.polling.async_base_polling import AsyncLROBasePolling
+
+
+_FINISHED = frozenset(["succeeded", "cancelled", "failed", "partiallysucceeded"])
+_FAILED = frozenset(["failed"])
+_SUCCEEDED = frozenset(["succeeded", "partiallysucceeded"])
+
+
+class TextAnalyticsAsyncLROPoller(AsyncLROBasePolling):
+    def __init__(self,
+        timeout=30,
```

Comment:
Just want to check - the timeout here will be used as the polling interval. The default is set to 30s but generally this value should be based off of what the service thinks is the right amount of time between polls. For example, in Form Recognizer we set this to 5s. If 30s is the correct time, then ignore me!

Reply:
@abhahn for more context: this also depends on how long endpoints take to run. If the service is slower, we want a longer timeout, up to you!

Reply:
I think for Analyze the time will be longer, but could probably set this to 5 or 10 seconds for Healthcare.  Since I'm using this class for polling for both services I can set the timeout when I create an instance of TextAnalyticsLROPoller in the clients.